data/transactions/logic/evalStateful_test.go

```diff
@@ -1388,187 +1530,210 @@ int 0x77
 ==
 &&
 `
-	delta := testApp(t, source, ep)
-	require.Empty(t, delta.GlobalDelta)
-	require.Len(t, delta.LocalDeltas, 1)
-
-	require.Len(t, delta.LocalDeltas[0], 2)
-	vd := delta.LocalDeltas[0]["ALGO"]
-	require.Equal(t, basics.SetUintAction, vd.Action)
-	require.Equal(t, uint64(0x77), vd.Uint)
-
-	vd = delta.LocalDeltas[0]["ALGOA"]
-	require.Equal(t, basics.SetBytesAction, vd.Action)
-	require.Equal(t, "ALGO", vd.Bytes)
-
-	// write same value without writing, expect no local delta
-	source = `int 0  // account
-byte 0x414c474f       // key
+		if ep.Proto.LogicSigVersion < directRefEnabledVersion {
+			source = strings.ReplaceAll(source, "txn Sender", "int 0")
```

Comment:
for some reason I don't like this but it does saves a lot of space.

Reply:
It's a bit gross. For new tests, I'm setting `sender` to either `int 0` or `txn Sender`, and pasting together the source I'll test. But to get old tests working...